packages/blockchain/src/index.ts

```diff
@@ -215,12 +214,6 @@ export default class Blockchain implements BlockchainInterface {
    * {@link BlockchainOptions}.
    */
   protected constructor(opts: BlockchainOptions = {}) {
-    // Throw on chain or hardfork options removed in latest major release to
-    // prevent implicit chain setup on a wrong chain
-    if ('chain' in opts || 'hardfork' in opts) {
-      throw new Error('Chain/hardfork options are not allowed any more on initialization')
-    }
```

Comment:
I removed this since it would be two major releases ago for develop, which I think is far enough to remove the legacy check, and TypeScript should guard downstream users against these options being used.

Reply:
👍